db/questions.yml

```diff
@@ -35,43 +35,155 @@ kitchen_problem:
       goto: describe_kitchen_damp
 
     - text: Taps
-      goto: kitchen_flooding_danger
+      goto: is_kitchen_tap_stuck_open
 
-    - text: Sockets
-
-    - text: Extractor fan
+    - text: Electrical
+      goto: kitchen_electrical_problem
 
     - text: Other
       goto: kitchen_other_problem
 
+is_kitchen_tap_stuck_open:
+  question: Is the tap stuck open?
+  answers:
+    - text: 'Yes'
+      page: emergency_contact
+
+    - text: 'No'
+
+kitchen_electrical_problem:
+  question: Is your problem one of these?
+  answers:
+    - text: Extractor fan
+      goto: is_fan_leaking_water
+
+    - text: Lights
+      goto: bathroom_lights_problem
+      todo: SHOULD GO SOMEWHERE ELSE?
+
+    - text: Smoke detector
+
+    - text: Sockets
+      page: electrical_hazard_warning
+
+other_electrical_problem:
+  question: Is your problem one of these?
+  answers:
+    - text: Light fitting
+      goto: water_leaking_into_light
+
+    - text: Light switches
+      page: electrical_hazard_warning
+
+    - text: Carbon monoxide detector
+
+    - text: Smoke detector
+
+    - text: Broken power socket
+      page: electrical_hazard_warning
+
 bathroom_problem:
   question: What is the problem in your bathroom?
   answers:
     - text: Toilet
       goto: toilet_problem
 
     - text: Taps
+      goto: is_bathroom_tap_stuck_open
+
     - text: Bath
+      goto: is_bath_leaking
+
     - text: Damp or mould
+      goto: describe_bathroom_damp
+
     - text: Basin
+      goto: is_basin_leaking_or_cracked
+
     - text: Extractor fan
+      goto: is_fan_leaking_water
 
     - text: Lights
-      goto: water_dripping_into_light
+      goto: bathroom_lights_problem
 
     - text: Something else
+      goto: bathroom_other_problem
+
+is_basin_leaking_or_cracked:
+  question: Is the basin leaking or cracked?
+  answers:
+    - text: 'Yes'
+
+    - text: 'No'
+      goto: is_basin_blocked
+
+is_basin_blocked:
+  question: Is the basin blocked?
+  answers:
+    - text: 'Yes'
+    - text: 'No'
+
+is_fan_leaking_water:
+  question: Is there water leaking into or from the fan?
+  answers:
+    - text: 'Yes'
+      page: emergency_contact
+
+    - text: 'No'
+
+is_bathroom_tap_stuck_open:
+  question: Is the tap stuck open?
+  answers:
+    - text: 'Yes'
+      page: emergency_contact
+
+    - text: 'No'
+
+is_bath_leaking:
+  question: Is your bath leaking?
+  answers:
+    - text: 'Yes'
+      page: emergency_contact
+
+    - text: 'No'
+      goto: is_bath_blocked_or_cracked
+
+is_bath_blocked_or_cracked:
+  question: Is your bath blocked or cracked?
+  answers:
+    - text: 'Yes'
+    - text: 'No'
+
+bathroom_lights_problem:
+  question: Is the problem with the...
+  answers:
+    - text: light fitting
```

Comment:
Should we standardise on Capitalised text?

Reply:
:+1: